testData/deviceReporter/installedSolutions.json

```diff
@@ -113,6 +113,10 @@
 
     {
         "id": "com.android.settings.system"
+    },
+
+    {
+        "id": "net.gpii.browserExtension"
```

Comment:
@kaspermarkus, I removed this, but it seems that the it's not picking up the solution anymore when i test with the windows repo.